src/tasks/harvest_groups.js

```diff
@@ -27,6 +28,18 @@ async function harvest () {
     allGroups.meetup.forEach(item => {
       console.log('Group Name:', item.name)
 
+      const rejected = isGroupRejected(item)
+
+      if (rejected) {
+        db.Group.destroy({
+          where: {
+            platform: 'meetup',
+            platform_identifier: `${item.id}`
+          }
+        })
+        return
+      }
+
```

Comment:
This results in operations like:

```
Executing (default): DELETE FROM "Groups" WHERE "platform" = 'meetup' AND "platform_identifier" = '32434124'
Executing (default): DELETE FROM "Groups" WHERE "platform" = 'meetup' AND "platform_identifier" = '21310669'
```

But I suppose we might also need an operation to remove any events related to that group.

1. When harvesting groups, we could remove any events from rejected groups

2. Or when harvesting events, we could remove any existing events which have been orphaned

I suppose 2. is preferable, because that would also remove events for groups which have completely disappeared from meetup.com

Is there any existing method to remove unwanted events? I don't see one.